### ISSUE TITLE
Algorithms

### ISSUE THREAD

**Issue by OfficialOxide** [econchick/new-coder#216]
It would be great if you added an algorithm tutorial on the site.

**Comment by merwok** [comment]
Hello and thanks for the suggestion!  Do you have ideas about a specific example?

**Comment by OfficialOxide** [comment]
Not exactly, but I was thinking an introduction to various kinds of sorting algorithms would be nice.

**Comment by mikaeilorfanian** [comment]
@OfficialOxide this repo(https://github.com/donnemartin/interactive-coding-challenges) has exactly what you need.

**Comment by OfficialOxide** [comment]
Thanks a lot. Will definitely look into this.
